src/Data/DataGetter.cpp

```diff
@@ -89,8 +90,8 @@ namespace SongDetailsCache {
         std::string line;
         while (std::getline(buf, line)) {
             if (line.starts_with("ETag: ")) {
-                auto tagStart = line.find_first_of('"') + 1;
-                auto tagLength = line.find_last_of('"') - tagStart;
+                auto tagStart = line.find_first_of('"');
```

Comment:
sorry what is this doing

Reply:
Server does not understand it without the quotes